src/components/fields/SchemaField.js

```diff
@@ -168,7 +168,9 @@ function SchemaFieldRender(props) {
   const FieldComponent = getFieldComponent(schema, uiSchema, idSchema, fields);
   const { DescriptionField } = fields;
   const disabled = Boolean(props.disabled || uiSchema["ui:disabled"]);
-  const readonly = Boolean(props.readonly || uiSchema["ui:readonly"]);
+  const readonly = Boolean(
+    props.readonly || uiSchema["ui:readonly"] || schema["readOnly"]
```

Comment:
```suggestion
    props.readonly || uiSchema["ui:readonly"] || props.schema.readOnly || schema.readOnly
```

To be consistent with the changes done in https://github.com/mozilla-services/react-jsonschema-form/commit/497620f95e6cb3a903996a60613b9a13ebd2238f